pkg/api/types.go

```diff
@@ -683,6 +683,14 @@ const (
 	AffinityTypeNone AffinityType = "None"
 )
 
+// LoadBalancerInfo defines whether the LoadBalancer is rewriting or
+// pass through
+type LoadBalancerInfo struct {
```

Comment:
It can move down to cloudprovider - it was in the API because I was going to bubble it through the whole system, but it turns out not to be necessary.